velox/docs/functions/spark/array.rst

```diff
@@ -15,6 +15,14 @@ Array Functions
 
         SELECT array(1, 2, 3); -- [1,2,3]
 
+.. spark::function:: arrays_zip(array(T), array(U),..) -> array(row(T,U, ...))
```

Comment:
nit: shall we adjust the order to put arrays_zip after array_xx functions?

Reply:
Fixed. Thanks!